### PR TITLE
PBI 41 & 42: create & read quiz

### PR DESCRIPTION
**Pull request by kanziebub**

### CODE REVIEW

File: src/Controllers/QuizController.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                'link_url' => 'required|string|max:65535',
+            ]);
+
+            if (! CourseUserHelper::isUserInCourse(
+                $user->id,
+                $request->input('course_id'),
+                CourseUserRole::TEACHER,
+            )) {
+                throw ValidationException::withMessages([
+                    'course_id' => 'Quiz not found',
```
Comment (AldiNFitrah) [inline src/Controllers/QuizController.php]: I think it should be `'course not found'`(?)

File: src/Migrations/2022_05_16_072252_create_quiz_table.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            $table->foreignId('course_id');
+            $table->foreignId('topic_id')->nullable();
+            $table->string('name', 255)->nullable();
+            $table->text('description')->nullable();
+            $table->dateTime('start_time')->nullable();
+            $table->dateTime('end_time')->nullable();
+            $table->integer('duration')->nullable();
+            $table->integer('point')->nullable();
+            $table->text('link_url')->nullable();
+            $table->foreignId('created_by');
```
Comment (AldiNFitrah) [inline src/Migrations/2022_05_16_072252_create_quiz_table.php:35]: don't forget to link this field to user table

**Review by Zsaschz — APPROVED** [review]
LGTM

**Review by AldiNFitrah — APPROVED** [review]
👌👌

**Pull request merged**